content/en/api/logs/logs.md

```diff
@@ -15,6 +15,11 @@ Send your logs to your Datadog platform over HTTP. Limits per HTTP request are:
 * Maximum size for a single log: 256kB
 * Maximum array size if sending multiple logs in an array: 500 entries
 
+All logs exceeding 256KB are accepted and truncated by the platform:
```

Comment:
```suggestion
All logs exceeding 256KB are accepted and truncated by Datadog:
```

Reply:
Should we replace `All logs` by `Any log entry in a batch` or something like this to make it more explicit?

Reply:
it has to include one-log-requests as well

Reply:
I could change to `Any log exceeding ..` though, WDYT ?

Reply:
Sounds like a great wording